README.md

```diff
@@ -530,9 +530,9 @@ If you need a more complex setup, rustup supports the convention used by
 the __curl__ program, documented in the ENVIRONMENT section of
 [its manual page][curlman].
 
-Note that some versions of `libcurl` apparently require you to drop the 
-`http://` or `https://` prefix in environment variables. For example, 
-`export http_proxy=proxy.example.com:1080` (and likewise for HTTPS). 
+Note that some versions of `libcurl` apparently require you to drop the
+`http://` or `https://` prefix in environment variables. For example,
+`export http_proxy=proxy.example.com:1080` (and likewise for HTTPS).
```

Comment:
While I dislike the trailing whitespace too, this hunk is unnecessary in the diff